libs/server-sdk/src/data_interfaces/destination/idestination.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <launchdarkly/data_model/descriptors.hpp>
+#include <launchdarkly/data_model/sdk_data_set.hpp>
+
+#include <string>
+
+namespace launchdarkly::server_side::data_interfaces {
+
+/**
+ * \brief IDestination represents a sink for data received by the
+ * SDK. A destination may be a database, local file, etc.
+ */
+class IDestination {
+   public:
+    /**
+     * \brief Initialize the destination with a base set of data.
+     * \param data_set The initial data received by the SDK.
```

Comment:
Is there a reason for the syntax change? from `@` to `\`. I guess it is canonical.

Reply:
CLion Nova is the reason. I failed to change the defaults. I will do so now, and can probably fix these all with a find/replace in the final feature PR.

Reply:
Well, I can't find the normal option to change the Doxygen format. Maybe it's not implemented yet.